packages/core/core/index.d.ts

```diff
@@ -2,7 +2,7 @@ import type {InitialParcelOptions, BuildEvent, BuildSuccessEvent, AsyncSubscript
 import type {FarmOptions} from '@parcel/workers';
 import type WorkerFarm from '@parcel/workers';
 
-declare class Parcel {
+export default class Parcel {
```

Comment:
Is the `declare` needed there?

Reply:
No; declarations need either an `export` or `declare`. I think you can include both, but it's redundant.